### DISCUSSION TITLE
Where can I see the source for the 22.11-ad1 branch of USD used for Maya-USD 0.23.0?

### DISCUSSION THREAD

**Discussion by pmolodo** [Autodesk/maya-usd#3003] [Q&A] [answered]
The [help docs for Maya-USD 0.23.0](https://help.autodesk.com/view/MAYAUL/2024/ENU/?guid=GUID-B4A60799-D557-4D68-8B87-A96E796BC7CC) mention that it's using "a version of USD that provides important security enhancements."  Clicking on the [support article](https://www.autodesk.com/support/technical/article/caas/tsarticles/ts/2aZO1BEEN1yOnBswYFuvwz.html) it links to mentions it's built with "USD 22.11-ad1", but I can't find the source for that anywhere.  There [doesn't seem to be a USD fork within the Autodesk organization](https://github.com/Autodesk?q=usd&type=all&language=&sort=).  Is there anywhere we can see this branch?

**Comment by zacarter** [comment]
I ran into the same issue today about USD 22.11-ad1 for USD 0.26 and bifrost 2.8, trying to get the USD nodes to be available in Bifrost. I can't find a build of USD 22.11-ad1

**Comment by wallworm** [comment]
We are working to provide visibility on this now. We'll post an update as soon as it is available.

**Reply by seando-adsk** [reply, marked as answer]
The source code to build the various USD versions has been pushed to the Autodesk Github USD fork (with tags). Please see the column "USD source for MayaUsd" in the table in section [Download and Build Pixar USD](https://github.com/Autodesk/maya-usd/blob/dev/doc/build.md#2-download-and-build-pixar-usd)
